### ISSUE TITLE
Setup failed for desmos chain.

### ISSUE THREAD

**Issue by tpmmthomas** [eco-stake/restake#222]
Might be a dump question. With the same mnemonic, I was able to set up the restake for osmosis chain, but was not able to set up on desmos chain.

By placing the corresponding `osmo...` address in the "botAddress" field in networks.json, It seems to give me a correct output:
```
Osmosis bot address is osmo145a2r0yurzkv43p2arhrwsqcy80pxg7guhadh2
Using REST URL https://lcd-osmosis.blockapsis.com
Using RPC URL https://osmosis.validator.network
You are using public nodes, script may fail with many delegations. Check the README to use your own
Running autostake
Bot balance is 0 uosmo
Bot balance is too low
```
But then when I did the same for desmos (of course using the corresponding `desmos...` address), it fails:
```
Desmos bot address is desmos145a2r0yurzkv43p2arhrwsqcy80pxg7gq5rdkq
Not an operator
Skipping
```

Any ideas why that would be the case? Thank you

**Comment by tombeynon** [comment]
It should work so long as the botAddress for your operator in networks.json is `desmos145a2r0yurzkv43p2arhrwsqcy80pxg7gq5rdkq`. Note this is different to the address Keplr gives you as we're not using the correct derivation path currently. A fix for this is planned in #78

**Comment by tpmmthomas** [comment]
thanks!

**Comment by tpmmthomas** [comment]
Sorry, one more problem. After I have changed my botaddress in networks.json, it gives me the following  output:
```
Running autostake
Bot balance is 0 udsm
Bot balance is too low
```
However, my wallet is actually FUNDED, is it an expected output as my operator info has not been merged to this repo yet, or it still means my setup is failed? Thanks a lot.

**Comment by tpmmthomas** [comment]
For your information, my latest botAddress is `desmos1cttl86ufne2qvctzdjtnunapr9y9qptt9chfra` (shown on keplr) while the botaddress used in this restake app, due to the wrong derivation path, is `desmos169n5p9xuhsnn85xunkg7sdmhtwqgwc3jhnzc0u`.

**Comment by tombeynon** [comment]
Hmm if it's the same address in your earlier messages it doesn't look funded currently https://morpheus.desmos.network/accounts/desmos145a2r0yurzkv43p2arhrwsqcy80pxg7gq5rdkq

Everything looks setup correctly otherwise, it's matching your operator etc. You just need to send some tokens to `desmos169n5p9xuhsnn85xunkg7sdmhtwqgwc3jhnzc0u`

**Comment by tpmmthomas** [comment]
oh I thought I should fund the keplr-shown address. Seems ok now, thanks so much.
One more question is that I now get the following output:
```
Desmos bot address is desmos169n5p9xuhsnn85xunkg7sdmhtwqgwc3jhnzc0u
Using REST URL https://rest.cosmos.directory/desmos
Using RPC URL https://rpc.mainnet.desmos.network
You are using public nodes, script may fail with many delegations. Check the README to use your own
Running autostake
Bot balance is 2900000 udsm
Finding delegators...

ERROR: Request failed with status code 504
```
Is the error again due to a failed setup or simply because no delegator has yet to set up restake with us? Thank you.

**Comment by tombeynon** [comment]
Interesting - 504 is a timeout, meaning it couldn't access the delegators URL. Strange, would expect a different error if it was because of the lack if delegations.

Does it happen everytime you run it? You can and should use your own nodes eventually, see README, but this should work fine in the meantime..

**Comment by tpmmthomas** [comment]
I guess I now got the "expected" output now, so thanks anyway:
```
Finding delegators...
...batch 1
Checking 98 delegators for grants...
...batch 1
Found 0 delegators with valid grants...
```

On another note, I am sorry to bother you for this as I know it isn't really an issue of your repo. But may I ask how I should set up for it to run using my local node? Given that I am running the full node in the same machine, and I have enabled api in the config of my full node,  my current setup in `networks.local.json` is "tcp://127.0.0.1:26657" for rpcUrl, and "tcp://127.0.0.1:1317" for restUrl. but it could not successfully search for my node, rather giving a message of `Failed to connect All promises were rejected`.
